kong/plugins/acme/schema.lua

```diff
@@ -1,11 +1,24 @@
 local typedefs = require "kong.db.schema.typedefs"
+local reserved_words = require "kong.plugins.acme.reserved_words"
 
 local CERT_TYPES = { "rsa", "ecc" }
 
 local RSA_KEY_SIZES = { 2048, 3072, 4096 }
 
 local STORAGE_TYPES = { "kong", "shm", "redis", "consul", "vault" }
 
+local function validate_namespace(namespace)
```

Comment:
let's make sure to communicate with koko team on this constraint

Reply:
OK, will inform the koko team on the JIRA ticket.